src/App.tsx

```diff
@@ -1,61 +1,38 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
 
 export const App = () => {
+  const enrichTodo = (todo: Todo) => ({
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId),
```

Comment:
Ensure that `usersFromServer` is an array of users with an `id` property. If `find` returns `undefined`, the `user` property in the enriched todo will be `undefined`, which might not be the desired behavior.